concordium-consensus/test-runners/deterministic/Main.hs

```diff
@@ -286,6 +286,7 @@ initialState = do
                 hconfig = NoHandler
                 config = SkovConfig gsconfig finconfig hconfig
             (_bsContext, _bsState) <- runLoggerT (initialiseSkov genData config) (logFor (fromIntegral bakerId))
+
```

Comment:
why this

Reply:
Good question. I have no recollection of touching this file :)